.github/workflows/build-download.yaml

```diff
@@ -56,4 +56,16 @@ jobs:
           uses: actions/upload-artifact@v2
           with:
             name: godotcord-binary-${{matrix.os}}
-            path: bin/godot.${{matrix.PLATFORM}}.opt.tools.${{matrix.EXTENSION}}
\ No newline at end of file
+            path: bin/godot.${{matrix.PLATFORM}}.opt.tools.${{matrix.EXTENSION}}
+
+        - name: Compile with target=debug
+          run: scons bits=64 target=debug -j2
+
+        - name: Modify binary after compilation
+          run: ${{matrix.AFTER_COMP_CMD}}
+
+        - name: Upload binary
+          uses: actions/upload-artifact@v2
+          with:
+            name: godotcord-binary-${{matrix.os}}-debug
+            path: bin/godot.${{matrix.PLATFORM}}.debug.opt.tools.${{matrix.EXTENSION}}

```

Comment:
Are you sure this is the name of the executable? When I compile it locally on windows with target=debug the resulting binary is called godot.windows.tools.64.exe

Reply:
Oh you're right. opt is only added when optimizations are enabled (in release builds with debug symbols stripped)

Reply:
Fixed

Reply:
My local compile also didn't have the .debug. in the name.

Reply:
Oh yup, my bad.
I had misunderstood what the path was for, but I fixed it. I thought it was to set the name of the executable at first.

Reply:
no problem